src/pages/patientView/clinicalTrialMatch/ClinicalTrialMatchTableOptions.tsx

```diff
@@ -116,96 +141,319 @@ class ClinicalTrialMatchTableOptions extends React.Component<
+                        >
+                            <Select
+                                options={this.countries.map(cnt => ({
+                                    label: cnt,
+                                    value: cnt,
+                                }))}
+                                isMulti
+                                name="CountrySearch"
+                                className="basic-multi-select"
+                                classNamePrefix="select"
+                                placeholder="Select countries..."
+                                onChange={(selectedOption: Array<any>) => {
+                                    const newStatuses = [];
+                                    if (selectedOption !== null) {
+                                        const statuses = selectedOption.map(
+                                            item => item.value
+                                        );
+                                        newStatuses.push(...statuses);
+                                    }
+                                    this.setState({
+                                        countryItems: newStatuses,
+                                    });
+                                }}
+                            />
+                        </div>
+                    </DefaultTooltip>
+
+                    <DefaultTooltip
+                        overlay={AGE_TOOLTIP}
+                        placement="topRight"
+                        trigger={['hover', 'focus']}
+                        destroyTooltipOnHide={true}
+                    >
+                        <div
+                            style={{
+                                display: 'block',
+                                marginLeft: '5px',
+                                marginBottom: '5px',
+                            }}
+                        >
+                            <Select
+                                options={this.ageList.map(age => ({
+                                    label: age,
+                                    value: age,
+                                }))}
+                                name="ageSearch"
+                                className="basic-select"
+                                classNamePrefix="select"
+                                placeholder="Select age..."
+                                onChange={(selectedOption: any) => {
+                                    var newStatuses = '';
+                                    if (selectedOption !== null) {
+                                        newStatuses = selectedOption.value;
+                                    }
+                                    this.setState({
+                                        age: +newStatuses,
+                                    });
+                                }}
+                            />
+                        </div>
+                    </DefaultTooltip>

```

Comment:
Although having the same style of select dropdowns for all user input isn't a bad idea, I would really prefer a simple number input field instead of a select for the age @lmahl

Reply:
Nice idea! I would also suggest to try using the clinical attribute `AGE` if present.